specs/sifive.com/PRCI/RESET/0.1.0/RESET_rtl.json5

```diff
@@ -0,0 +1,42 @@
+{
+  abstractionDefinition: {
+    vendor: 'sifive.com',
+    library: 'PRCI',
+    name: 'RESET_rtl',
+    version: '0.1.0',
+    busType: {
+      vendor: 'sifive.com',
+      library: 'PRCI',
+      name: 'RESET',
+      version: '0.1.0',
+    },
+    ports: {
+      RESET: {
+        description: 'reset signals',
+        wire: {
+          isReset: true,
+          onMaster: {direction: 'out', presence: 'required'},
+          onSlave:  {direction: 'in', presence: 'required'}
+        }
+      }
+    },
+    props: {
+      ResetPolarity: {
+          type: "string",
+	  enum: ["ActiveHigh", "ActiveLow"],
+      },
+      ResetType: {
+	  type: "string",
+	  enum: ["Synchronous", "Asynchronous", "Full Asynchronous"]
```

Comment:
What is the difference between `Asynchronous` and `Full Asynchronous` ?

Reply:
Hm, I have the same question. Full Asynchronous in our terms is more of a control to the design to make it have all its registers asynchronously reset, but from a wire standpoint it shouldn't need to know whether it is Asynchronous or Full Asynchronous. 

And just to be clear on the meaning of the signal, since Asynchronous and Synchronous are overloaded in the realm of resets... we mean that this is how the flops downstream are using the reset, right? Not actually its relationship to the its paired clock. The two CAN be considered somewhat orthogonal.

Given that meaning, I think a better additional option here is "Mixed" because a wire coming in could be used for both Synchronous and Asynchronous flops (which is obviously problematic to most CDC lint tools... but some IP are going to do it.)

Reply:
I'm referring to the SiFive reset spec at https://github.com/sifive/arch-specs/blob/master/reset.adoc .   

Yes, It's referring to the type of reset structure that the downstream flops will use (async input vs synchronous mux esssentially).  Since we plan on distributing reset asynchronously from the source, we still need to know async vs sync because that controls what the synchronizer must do.  Async reset synchronizer must still allow synchronous deassertion but async assertion.  Sync reset synchronizer must be synchronous in both cases.

Note that I put all three options here because I couldn't convince myself that we would never need to know async vs full async.  We probably don't, but I figured the placeholder was not a bad thing ;o)